Controller/CRUDController.php

```diff
@@ -362,7 +362,6 @@ public function editAction($id = null)
         $form->handleRequest($request);
 
         if ($this->getRestMethod() == 'POST') {
-            $form->submit($request);
```

Comment:
I do not removed because @greg0ire add it before (https://github.com/sonata-project/SonataAdminBundle/commit/77b4c16f58858df7d0cf0ffcca0792be81c64e00#diff-b1f1981b1ed9490b32c9bc83360655c3R380) and how @Soullivaneuh said it is going to be removed.

Although, I am going to change to do like symfony recommend.